CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ target_compile_definitions(ot-config INTERFACE
     OPENTHREAD_CONFIG_FILE="openthread-core-efr32-config.h"
     OPENTHREAD_PROJECT_CORE_CONFIG_FILE="openthread-core-efr32-config.h"
     OPENTHREAD_CORE_CONFIG_PLATFORM_CHECK_FILE="openthread-core-efr32-config-check.h"
-    OPENTHREAD_CONFIG_NCP_UART_ENABLE=1
+    OPENTHREAD_CONFIG_NCP_HDLC_ENABLE=1
```

Comment:
Is this line redundant with the define in [src/src/openthread-core-efr32-config.h](https://github.com/openthread/ot-efr32/blob/23c69b7302682666b93fa253128a9c3ffd48472d/src/src/openthread-core-efr32-config.h)?

Reply:
Could be, I just made the rename changes in place wherever I found it.  Let me test it with it removed.

Reply:
Yes, it was redundant. Removed the line.  Thanks for catching that!